Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -2452,6 +2452,23 @@ bool AccessibilityObject::ignoredFromModalPresence() const
     if (modalNode->document().frame() != this->frame())
         return false;
     
+    // Some objects might be outside of a modal, but are linked to elements inside of it. Don't ignore those.
+    for (RefPtr ancestor = this; ancestor; ancestor = ancestor->parentObject()) {
+        for (auto& controller : ancestor->controllers()) {
+            if (RefPtr controllerObject = dynamicDowncast<AccessibilityObject>(controller)) {
```

Comment:
I think we should just `downcast<AccessibilityObject>`. This should never be an isolated object